### ISSUE TITLE
Gauge addition

### ISSUE THREAD

**Issue by bradmowery-mac** [ClutchEngineering/Sidecar#55]
It would be nice to have a gauge view for both phone and CarPlay so I can monitor additional information that I don’t get on my dash

**Comment by jverkoey** [comment]
In this scenario are you having your iPhone and CarPlay both mounted to your dash?

**Comment by aditya369007** [comment]
This sounds like an amazing addition! I currently have my phone on a MagSafe mount right beside my CarPlay screen, having the option to display more gauges will be amazing. 

Another thing I can think of is if (for example) I can have a gauge that corresponds to RPM displaying on my phone, I can put my phone right below my windscreen and it becomes a makeshift HUD.

**Comment by DantasticFour** [comment]
Agreed. An example gauge use case could be TPMS context about specific tire & reported pressure, whereas the dash only reports one or more are problematic.

**Comment by bradmowery-mac** [comment]
Tpms data and temps gauges sounds like exactly what I was hoping for!Sent from my iPhoneOn Dec 12, 2024, at 10:32 AM, DantasticFour ***@***.***> wrote:﻿
Agreed. An example gauge use case could be TPMS context about specific tire & reported pressure, whereas the dash only reports one or more are problematic.
